.github/workflows/playwright.yml

```diff
@@ -182,14 +188,18 @@ jobs:
           "install + policy"*) TFILTER="\/[09]";;
           "install + all"*)    TFILTER="\/[0-9]";;
         esac
-        # Custom filter to match file name or test (\/90 -g 'Privileged')
-        [ -n "${{inputs.testfilter}}" ] && TFILTER="${{inputs.testfilter}}"
 
+        # Custom filter to match file name or test
+        [ -n "$TESTFILTER" ] && TFILTER="$TESTFILTER"
+
+        # List tests and execute
+        echo yarn playwright test ${TFILTER:-} -x
         yarn playwright test ${TFILTER:-} -x
       env:
+        FLEET: ${{ github.event.schedule == '0 3 * * *' || inputs.fleet }}
+        TESTFILTER: ${{inputs.testfilter}}
         RANCHER_URL: https://${{env.RANCHER_FQDN}}
         ORIGIN: ${{ env.KUBEWARDEN }}
-        # PLAYWRIGHT_HTML_REPORT: test-report
```

Comment:
What's the benefit of removing the html report, do we still get the playwright-report artifact with this removed?

Reply:
Yes, doc says: `By default, report is written into the playwright-report folder in the current working directory. One can override that location using the PLAYWRIGHT_HTML_REPORT`

Custom folder was used because I did:
1. run all installation tests (write into playwright-report folder)
2. test all the policies (rewrite playwright-report folder)

Now we schedule all at once.